HelpMyStreetFE/HelpMyStreetFE/src/js/profile/requests.js

```diff
@@ -40,4 +42,35 @@ export function initialiseRequests() {
       $(`#job-${id} .job__detail`).slideToggle();
     });
   });
+
+
+    $('.accept-request').click(function (evt) {
+        if ($(this).attr("data-accepted") != "true") {
+        let popupMessage = '<p>Once you accept a task, you are responsible for working with the person who made the request (and / or the person who needs the help - if it was made on behalf of someone else) to get it done.</p>'
```

Comment:
Use string literals to avoid have to put `+` to spread over multiple lines.
Also would be better if we didn't have to specify HTML in JavaScript at all. Bearing in mind this HTML is static for the popup "type" could we specify it where the partial is used in the cshtml?